src/test/java/DemoqaTests.java

```diff
@@ -0,0 +1,119 @@
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+import static org.openqa.selenium.By.cssSelector;
+
+public class DemoqaTests {
+    public static final String BASIC_URL = "https://demoqa.com/";
+    public static final String SEARCH_BOX = "searchBox";
+    public static final String FIRST_CHILD = "div.rt-tr-group div.rt-tr.-odd div.rt-td:first-child";
+    public static final String ADD_NEW_RECORD_BUTTON = "addNewRecordButton";
+    public static final String MODAL_TITLE = "registration-form-modal";
+    public static final String DOUBLE_CLICK_BTN = "doubleClickBtn";
+    public static final String RIGHT_CLICK_BTN = "rightClickBtn";
+    public static final String DOUBLE_CLICK_MESSAGE = "doubleClickMessage";
+    public static final String RIGHT_CLICK_MESSAGE = "rightClickMessage";
+    public static final String IFRAME_TITLE_RE_CAPTCHA = "iframe[title='reCAPTCHA']";
+    public static final String RECAPTCHA_ANCHOR = "recaptcha-anchor";
+    public static final String REGISTER_BUTTON = "register";
+    public static final String USER_NAME_VALUE = "userName-value";
+    public static final String BUTTON_LOGIN = "button#login";
+    WebDriver driver;
+
+    @BeforeEach
+    void setup() {
+        driver = new FirefoxDriver();
+        driver.manage().window().fullscreen();
```

Comment:
fullscreen is not the best option for testing. it is better to set specific resolution